src/Sentry/SentrySdk.cs

```diff
@@ -29,6 +28,14 @@ internal static IHub InitHub(SentryOptions options)
     {
         options.SetupLogging();
 
+#if NETSTANDARD
+        if (AotHelper.IsDotNetNative)
+        {
+            options.LogWarning("Sentry doesn't yet support .NET Native compilation. The SDK will be disabled.");
```

Comment:
@bitsandfoxes I'm not sure whether `options.LogWarning` is the right approach... it can easily go unnoticed. Perhaps we should be throwing an exception here instead (but one with a self explanatory error message)?

Reply:
Yeah. I think we agreed to throw. The reasons were
1. It's in init. So fail fast.
2. It's going to crash anyway. So I'd rather have 1. happen.